src/box/lua/console.c

```diff
@@ -692,6 +692,13 @@ tarantool_lua_console_init(struct lua_State *L)
 	lua_pushcclosure(L, lbox_console_readline, 1);
```

Comment:
> has no clear benefits

I disagree here. This approach allows to preview a copy-pasted multiline command or a list of commands before actually execute them. The problem here is that the code for reading input should be support this mode on its side: split received text per lines.

I think it would be good to support it in a future, just don't want to insist on doing so within this task. It is okay to disable the mode within this PR, because we actually don't support it:

* Lua statements will not show results.
* SQL language in console will give parsing error.
* Backslashes at end of lines will not be removed.

Reply:
You are right, reworded the commit msg.